addons/mass_mailing/models/mail_mail.py

```diff
@@ -64,8 +73,7 @@ def _send_prepare_body(self):
 
             # generate tracking URL
             tracking_url = self._get_tracking_url()
-            if tracking_url:
-                body = tools.append_content_to_html(body, tracking_url, plaintext=False, container_tag='div')
+            body = tools.append_content_to_html(body, f'<img src="{tracking_url}" alt=""/>', plaintext=False, container_tag='div')
```

Comment:
Using fstring makes code harder to understand. Do a simple %s when having 1 parameter.